.github/workflows/_build-python-wheel-extension.yml

```diff
@@ -8,6 +8,10 @@ on:
         default: 'PROJECT_NAME'
         required: true
         type: string
+      python_version:
```

Comment:
this .yml is for non-pure Python - use the `pyython-version` from the main `_build-wheel-..yml` file.